ue4docker/infrastructure/WindowsUtils.py

```diff
@@ -116,19 +98,18 @@ def isInsiderPreview() -> bool:
 		return WindowsUtils.getWindowsBuild() > WindowsUtils._latestReleaseBuild
 
 	@staticmethod
-	def getReleaseBaseTag(release: str) -> str:
+	def getHostBaseTag() -> str:
 		'''
-		Retrieves the tag for the Windows Server Core base image matching the specified Windows 10 / Windows Server release
+		Retrieves the tag for the Windows Server Core base image matching the host Windows system
 		'''
 
-		# For Windows Insider preview builds, build the latest release tag
-		if WindowsUtils.isInsiderPreview():
-			return WindowsUtils._validTags[-1]
+		hostBuild = WindowsUtils.getWindowsBuild()
 
-		# This lookup table is based on the list of valid tags from <https://hub.docker.com/r/microsoft/windowsservercore/>
-		return {
-			'1809': 'ltsc2019',
-		}.get(release, release)
+		# Note that we fallback to hostBuild to avoid pretending that everything is fine
+		# if we're running on an unsupported Windows build (like, EOLed or Insider Preview).
+		# This fallback will later be rejected when ue4-docker checks whether it is valid.
+		# Users of unsupported Windows builds can still use ue4-docker by using -basetag option with a known value.
+		return WindowsUtils._tagsByBuildNumber.get(hostBuild, str(hostBuild))
```

Comment:
@TBBle This particular change is a bit questionable, maybe you have some comments on it?

Reply:
Perhaps we could just return `None` and catch it elsewhere, requiring that the user provide an explicit base-tag (and possibly force process isolation if they know it's safe) They'd still get warnings, which seems appropriate for "You have to know what you're doing at this point".

Alternatively, we could return the full version (10.0.XXXX.YY) which would be a valid tag if there's a matching image published (should be the case for all non-Insider builds), and hence is somewhat future-proof, but might be surprising to users who would have expected failure in that case, and certainly would have expected a floating tag (SAC-style or `ltsc20xx`) for the resulting image.